### DISCUSSION TITLE
Welcome to the ML-Nexus Discussion Hub! 💬

### DISCUSSION THREAD

**Discussion by UppuluriKalyani** [UppuluriKalyani/ML-Nexus#8] [Announcements]
### Welcome to the ML-Nexus Discussion Hub! 💬

Hey there! 👋 Welcome to the discussion section of ML-Nexus. This is the place to share your thoughts, brainstorm new ideas, and ask questions. We're here to collaborate and help each other build amazing machine learning projects!

**Here’s how you can use this space:**

1. **Ask Questions**: Got a question about ML-Nexus or machine learning in general? Ask away! The community is here to help.

2. **Project Ideas**: Have a cool project idea but not sure how to start? Share it here, and let’s brainstorm together.

3. **Feedback & Suggestions**: Got feedback on the repository or ideas for improvement? Drop your thoughts, and we’ll discuss how to implement them.

4. **Collaborations**: Looking for someone to collaborate on a project? Post here, and team up with other contributors.

5. **New Topic Discussions**: Dive into specific machine learning areas, such as:
   - Natural Language Processing (NLP)
   - Computer Vision
   - Neural Networks
   - Data Preprocessing and Feature Engineering
   - Any other ML topic that excites you!

6. **Showcase Your Work**: Proud of something you built? Post your project and results here to inspire others.

**Let’s Keep It Friendly**:  
Please remember to be respectful and supportive. We're here to learn and grow together!

Feel free to start a new discussion and let your creativity flow. We're excited to see what you bring to the table! 🚀

**Comment by SrujanaBora** [comment]
How do I start contributing to ML-Nexus?

**Reply by UppuluriKalyani** [reply]
> How do I start contributing to ML-Nexus?

First, fork the ML-Nexus repository and clone it to your local machine. Check the open issues for tasks or suggest your own idea. Once you have a solution, create a pull request (PR) with your code and include any results or outputs from your project. Make sure to follow the contribution guidelines!

**Comment by DebanKsahu** [comment]
Do implementing any paper for practice count as a project. I am currently working in SkimLit and I want to give it a frontend using streamlit. Could I use as a project here??

**Reply by UppuluriKalyani** [reply]
> Do implementing any paper for practice count as a project. I am currently working in SkimLit and I want to give it a frontend using streamlit. Could I use as a project here??
 Yeah I would love include SkimLit as an NLP project in ML-Nexus. Please feel free to start an issue!

**Reply by DebanKsahu** [reply]
Thanks for clarification. Is there any deadline or we can submit within 31 October

**Reply by UppuluriKalyani** [reply]
> Thanks for clarification. Is there any deadline or we can submit within 31 October

Yes you should submit this within time period

**Comment by UppuluriKalyani** [comment]
Welcome guyyys!!..Happy to see you all here..and don't forget to star the repo, let's connect and follow eachother 🙃...all the best!

**Comment by BhavyaFattania** [comment]
this project comes under hactoberfest 2024?

**Reply by UppuluriKalyani** [reply]
> this project comes under hactoberfest 2024?

Yeah we collaborated with hacktoberfest

**Comment by DebanKsahu** [comment]
i have some issues while performing git push 
**
(tf217) (base) root@LAPTOP-45IOM653:~/My_Projects/GSSOc 2024-extd/ML-nexux_project(skimlit)/ML-Nexus# git push origin fix-issue-19 --verbose
Pushing to https://github.com/DebanKsahu/ML-Nexus.git
Enumerating objects: 26, done.
Counting objects: 100% (26/26), done.
Delta compression using up to 16 threads
Compressing objects: 100% (21/21), done.
Writing objects: 100% (24/24), 567.13 MiB | 18.65 MiB/s, done.
Total 24 (delta 3), reused 0 (delta 0), pack-reused 0
POST git-receive-pack (594682321 bytes)
error: RPC failed; HTTP 400 curl 92 HTTP/2 stream 1 was not closed cleanly: CANCEL (err 8)
send-pack: unexpected disconnect while reading sideband packet
fatal: the remote end hung up unexpectedly
Everything up-to-date
**

could anybody help

**Reply by UppuluriKalyani** [reply]
> i have some issues while performing git push ** (tf217) (base) root@LAPTOP-45IOM653:~/My_Projects/GSSOc 2024-extd/ML-nexux_project(skimlit)/ML-Nexus# git push origin fix-issue-19 --verbose Pushing to https://github.com/DebanKsahu/ML-Nexus.git Enumerating objects: 26, done. Counting objects: 100% (26/26), done. Delta compression using up to 16 threads Compressing objects: 100% (21/21), done. Writing objects: 100% (24/24), 567.13 MiB | 18.65 MiB/s, done. Total 24 (delta 3), reused 0 (delta 0), pack-reused 0 POST git-receive-pack (594682321 bytes) error: RPC failed; HTTP 400 curl 92 HTTP/2 stream 1 was not closed cleanly: CANCEL (err 8) send-pack: unexpected disconnect while reading sideband packet fatal: the remote end hung up unexpectedly Everything up-to-date **
> 
> could anybody help

You’re encountering a push error due to large files exceeding Git's default buffer size. Try increasing the buffer size using git config --global http.postBuffer 524288000. Alternatively, switch from HTTPS to SSH for a more stable connection, or consider using Git LFS to manage large files efficiently. Finally, ensure your internet connection is stable before retrying.

**Reply by DebanKsahu** [reply]
I tried with git buffer but not solved so i commited and merged seperately to my fork repo and it merged. Now i am test if all working or not.

**Reply by DebanKsahu** [reply]
(tf217) (base) root@LAPTOP-45IOM653:~/ML-Nexus/Natural Language Processing/SkimLit/SkimLit_Models# git lfs push --all origin
batch response: @DebanKsahu can not upload new objects to public fork DebanKsahu/ML-Nexus 

I am getting a error like this during using git lfs.

**Reply by UppuluriKalyani** [reply]
> (tf217) (base) root@LAPTOP-45IOM653:~/ML-Nexus/Natural Language Processing/SkimLit/SkimLit_Models# git lfs push --all origin batch response: @DebanKsahu can not upload new objects to public fork DebanKsahu/ML-Nexus
> 
> I am getting a error like this during using git lfs.

Hey, just a heads-up—when pushing large files, it's better to avoid doing it all at once. Try splitting the files and pushing smaller batches. Also, I recommend using SSH instead of HTTPS for such operations as it’s more reliable for large data transfers.

**Comment by UppuluriKalyani** [comment]
@DebanKsahu Btw which issue you are working on

**Reply by DebanKsahu** [reply]
Issue 19 which is to add new project

**Reply by DebanKsahu** [reply]
Could i work with a new forked repo. I think I really messed up this repo so I already saved files necessary files of this repo and think to start with a new forked repo

**Reply by UppuluriKalyani** [reply]
What??..that's not an issue to solve. It's just like guidelines how to keep your project while submission.

**Reply by DebanKsahu** [reply]
Actually it's my first time handling project in GitHub but didn't expect I will messed up this much.

**Reply by UppuluriKalyani** [reply]
> Actually it's my first time handling project in GitHub but didn't expect I will messed up this much.

Don't worry, First make yourself familiar with it how to do contributing, please read the guidelines before you start, just go through readme file of this repo and start working.

**Comment by UppuluriKalyani** [comment]
@DebanKsahu please make new issue (idea) and start working on it after it assigned to you

**Comment by DebanKsahu** [comment]
batch response: @DebanKsahu can not upload new objects to public fork DebanKsahu/ML-Nexus                                                                               
Uploading LFS objects:   0% (0/2), 0 B | 0 B/s, done.
error: failed to push some refs to 'https://github.com/DebanKsahu/ML-Nexus.git'
I am facing this issue when i try to push my commits into the forked repo after using git lfs for larger models, but when i push them into my private repo i didn't faced this issue. Could anyone explain about this.

**Reply by DebanKsahu** [reply]
@UppuluriKalyani you haven't replied yet ?

**Comment by UppuluriKalyani** [comment]
@DebanKsahu delete existing fork and fork the repo again and try to push your changes and make sure you should not push large size files

**Reply by DebanKsahu** [reply]
Ok then.

**Comment by rakheshkrishna2005** [comment]
I am trying to add a contribute a new project to the repository. I have raised an issue. But the github-actions bot is automatically closing the issue as completed. Please reopen the issue and assign it to me so that I can contribute.

Issue #396

**Comment by sidgun8** [comment]
Hey, 

I'm Siddharth, I am very new to open source contributions, I have a good hold of Data Analytics and Time series Analysis with Python and found this project to be a great aligning fit to learn and upskill. I was a little tight on other commitments the last few weeks and the extension came like a boon. I wanted to make at least a few contributions before the deadlines up and wanted to know on how I could get a issue alloted? I've forked and cloned this to my local machine and wanted to contribute to any time series analysis project available(eg the new zealand migration one). Is working on any dataset u find and once results produced giving a submission request here considered a conribution? And can I work on as issue allloted to someone else? Would that be considered a contribution?

**Reply by UppuluriKalyani** [reply]
Hey Siddharth, Check the Issues tab for unassigned issues or if you have any ideas please raise an issue. Once done, submit a well-documented PR for review!
